pytest_repeat.py

```diff
@@ -37,6 +37,8 @@ def __pytest_repeat_step_number(request):
 
 def pytest_generate_tests(metafunc):
     count = metafunc.config.option.count
+    if hasattr(metafunc.function, 'repeat'):
+        count = int(metafunc.function.repeat.args[0])
```

Comment:
this will be incorrect for markers coming from parameterization

Reply:
@RonnyPfannschmidt Thanks for your feedback! Would you mind elaborating a bit? This is my first `pytest` contribution and I still don't know much about the project. If you could explain the case under which this implementation will fail or provide a pointer on how you would implement it instead, that would be great. :innocent:

Reply:
Recent pytest has marker Objekt access via metafunc.node

Im currently on mobile and may fail to follow up

Reply:
I do not see any `.node` attribute in `metafunc`. Also haven't found any `metafunc.node` occurrences in GitHub. :sweat_smile: 

Could you provide a case that fails with the current implementation? Tests seem to pass alright for `pytest` 2 and 3 and Python 2 and 3. :blush:

Reply:
my bad, i thought of the wrong api, i made an example using the correct api below
thanks for the patience

the problem would be a test like

```python
import pytest

@pytest.mark.parametrize('foo', [pytest.param(1, marks=pytest.mark.repeat(3)), 2])
def test_fun(foo):
 pass
```

while writing i realized that this doesn't fit the current parameterization scheme to begin with, so you shouldn't handle